docs/modules/ROOT/pages/installation/registry/registry.adoc

```diff
@@ -41,7 +41,7 @@ The changes will be immediately reconciled and the operator will be able to push
 == Container registry requirements
 Each platform may have its default registry of choice. And each container registry may have a slight different configuration. Please, be aware that we won't be able to support all the available solutions.
 
-The only requirement we have is that the registry must be able to produce/consume images with the following tagging convention: `<registry-host>[:<registry-port>]/<k8s-namespace>/kit-<hash-code>@sha256:<sha256-code>`, ie `10.110.251.124/default/kit-ck0612dahvgs73ffe5g0@sha256:3c9589dd093b689aee6bf5c2d35aa1fce9d0e76d5bb7da8b61d87e7a1ed6f36a`.
+The only requirement we have is that the registry must be able to produce/consume images with the following tagging convention: `<spec.build.registry.address[:port]>/<spec.build.registry.organization>/camel-k-kit-<hash-code>@sha256:<sha256-code>`, ie `10.110.251.124:5000/camel-k/camel-k-kit-ck0612dahvgs73ffe5g0@sha256:3c9589dd093b689aee6bf5c2d35aa1fce9d0e76d5bb7da8b61d87e7a1ed6f36a`.
```

Comment:
We need to fix `camel-k-kit` for sure. As for the other parameter I'd leave them how they are, and then I'd prefer a section above specifying where the registry address is taken and how the organization is calculated. BTW, this is explained in https://camel.apache.org/camel-k/next/installation/registry/registry.html#organization as well.